src/components/Synthetics/GmSwap/GmSwapBox/GmSwapBox.tsx

```diff
@@ -116,7 +116,7 @@ export function GmSwapBox(p: GmSwapBoxProps) {
       {operation === Operation.Deposit || operation === Operation.Withdrawal ? (
         <GmSwapBoxDepositWithdrawal
           selectedMarketAddress={selectedMarketAddress}
-          glvMarket={marketInfo as GlvMarketInfo}
+          glvMarket={marketInfo as GlvInfo}
```

Comment:
confusing place, it seems like it's actually a market info, isn't it?

Reply:
This prop is unused, will remove